src/nft/index.ts

```diff
@@ -10,14 +12,36 @@ import {
     EthDeployErc721,
     EthMintErc721,
     EthMintMultipleErc721,
-    EthTransferErc721
+    EthTransferErc721,
+    MintMultiToken,
+    MintMultiTokenBatch,
+    TransferMultiToken,
+    TransferMultiTokenBatch,
+    CeloTransferMultiToken,
+    CeloTransferMultiTokenBatch
 } from '../model';
-import {CeloUpdateCashbackErc721} from '../model/request/CeloUpdateCashbackErc721';
-import {UpdateCashbackErc721} from '../model/request/UpdateCashbackErc721';
+import { CeloUpdateCashbackErc721 } from '../model/request/CeloUpdateCashbackErc721';
```

Comment:
Multi token stuff should not be inside nft/index.ts, but in its own directory multitoken/index.ts